test/integration/gnosisSafeNestedSafes.spec.ts

```diff
@@ -0,0 +1,160 @@
+import { expect } from "chai";
+import hre, { deployments, waffle } from "hardhat";
+import "@nomiclabs/hardhat-ethers";
+import { AddressZero } from "@ethersproject/constants";
+import { parseEther } from "@ethersproject/units";
+import { defaultAbiCoder } from "@ethersproject/abi";
+import { getSafeWithOwners, getCompatFallbackHandler } from "../utils/setup";
+import { buildSignatureBytes, signHash, executeContractCallWithSigners} from "../../src/utils/execution";
+
+
+describe("NestedSafes", async () => {
+    const [user1, user2, user3, user4, user5] = waffle.provider.getWallets();
+    
+    const setupTests = deployments.createFixture(async ({ deployments }) => {
+        await deployments.fixture();
+        const signLib = await (await hre.ethers.getContractFactory("SignMessageLib")).deploy();
+        const handler = await getCompatFallbackHandler()
+        const safe1 = await getSafeWithOwners([user1.address, user2.address], 2, handler.address)
+        const safe2 = await getSafeWithOwners([user3.address, user4.address], 2, handler.address)
+        const parentSafe = await getSafeWithOwners([safe1.address, safe2.address], 2, handler.address)
+        const handlerSafe1 = handler.attach(safe1.address)
+        const handlerSafe2 = handler.attach(safe2.address)
+        return {
+            safe1,
+            safe2,
+            parentSafe, 
+            handlerSafe1,
+            handlerSafe2,
+            signLib
+        }
+    })
+
+    it('should use EIP-1271 (contract signatures)', async () => {
+        const { safe1, safe2, parentSafe,handlerSafe1,handlerSafe2 } = await setupTests()
```

Comment:
How is possible that linter passed?

Reply:
Because the linter is only checking the .sol files.